pytorch_lightning/overrides/fairscale.py

```diff
@@ -0,0 +1,37 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import platform
+
+from pytorch_lightning.utilities import _module_available
+
+if platform.system() != "Windows" and _module_available('fairscale.nn.data_parallel.sharded_ddp'):
```

Comment:
```suggestion
FAIRSCALE_SHARDED_AVAILABLE = _module_available('fairscale.nn.data_parallel.sharded_ddp'):
if FAIRSCALE_SHARDED_AVAILABLE:
```
maybe move the `FAIRSCALE_SHARDED_AVAILABLE` to utils init as others